src/generators/jsx/utils/buildContent.mjs

```diff
@@ -0,0 +1,199 @@
+'use strict';
+
+import { h as createElement } from 'hastscript';
+import { u as createTree } from 'unist-builder';
+import { SKIP, visit } from 'unist-util-visit';
+
+import createQueries from '../../../utils/queries/index.mjs';
+import { createJSXElement } from './ast.mjs';
+import {
+  ICON_SYMBOL_MAP,
+  STABILITY_LEVELS,
+  CHANGE_TYPES,
+} from '../constants.mjs';
+import { DOC_NODE_BLOB_BASE_URL } from '../../../constants.mjs';
+import { enforceArray, sortChanges } from '../../../utils/generators.mjs';
+import { buildMetaBarProps } from './buildBarProps.mjs';
+
+/**
+ * Transforms a stability node into an AlertBox JSX element
+ *
+ * @param {import('mdast').Blockquote} node - The stability node to transform
+ * @param {number} index - The index of the node in its parent's children array
+ * @param {import('unist').Parent} parent - The parent node containing the stability node
+ * @returns {[typeof SKIP]} Visitor instruction to skip the node
+ */
+function transformStabilityNode(node, index, parent) {
+  const { data } = node;
+  parent.children[index] = createJSXElement('AlertBox', {
+    children: data.description,
+    level: STABILITY_LEVELS[data.index],
+    title: data.index,
+  });
+  return [SKIP];
+}
+
+/**
+ * Creates a history of changes for an API element
+ *
+ * @param {ApiDocMetadataEntry} entry - The metadata entry containing change information
+ * @returns {import('unist').Node|null} JSX element representing change history or null if no changes
+ */
+function createChangeElement(entry) {
+  // Collect changes from version fields (added, deprecated, etc.)
+  const changeEntries = Object.entries(CHANGE_TYPES)
+    .filter(([field]) => entry[field])
+    .map(([field, label]) => {
+      const versions = enforceArray(entry[field]);
```

Comment:
Maybe call another map to apply the enforceArray, so that the last map can just return `({ versions, label: ... })`